scripts/steam.zs

```diff
@@ -1,33 +1,134 @@
 mods.immersivetechnology.Boiler.addFuel(<liquid:diesel> * 40, 1, 10);
 mods.immersivetechnology.Boiler.addFuel(<liquid:gasoline> * 110, 1, 10);
-mods.immersivetechnology.Distiller.addRecipe(<liquid:distwater> * 15360, <liquid:water> * 15360, <immersivetech:material>, 8000, 20, 0.1);
-mods.immersivetechnology.SolarTower.addRecipe(<liquid:low_pressure_steam> * 61440, <liquid:distwater> * 61440, 20);
+
+mods.immersivetechnology.Distiller.addRecipe(<liquid:condensate_water> * 3840, <liquid:water> * 3840, <immersivetech:material>, 8000, 20, 0.1);
+
+mods.immersivetechnology.SolarTower.addRecipe(<liquid:low_pressure_steam> * 61440, <liquid:condensate_water> * 960, 20);
+
 mods.immersivetechnology.SteamTurbine.addFuel(<liquid:low_quality_steam> * 6144, <liquid:low_pressure_steam> * 3072, 1);
-mods.immersivetechnology.HighPressureSteamTurbine.addFuel(<liquid:exhauststeam> * 6144, <liquid:high_pressure_steam> * 1536, 1);
+
+mods.immersivetechnology.HighPressureSteamTurbine.addFuel(<liquid:exhaust_steam> * 6144, <liquid:high_pressure_steam> * 1536, 1);
+
+mods.immersivetechnology.Radiator.addRecipe(<liquid:condensate_water> * 25, <liquid:exhaust_steam> * 400, 64);
+mods.immersivetechnology.Radiator.addRecipe(<liquid:condensate_water> * 25, <liquid:low_quality_steam> * 800, 27);
+
 mods.immersivetechnology.GasTurbine.addFuel(<liquid:fluegas> * 1000, <liquid:diesel> * 400, 10);
 mods.immersivetechnology.GasTurbine.addFuel(<liquid:fluegas> * 1000, <liquid:biodiesel> * 600, 10);
 mods.immersivetechnology.GasTurbine.addFuel(<liquid:fluegas> * 1000, <liquid:gasoline> * 1100, 10);
+
 mods.immersiveengineering.Refinery.addRecipe(<liquid:biodiesel> * 20, <liquid:plantoil> * 10, <liquid:ethanol> * 10, 2048);
+
 mods.immersivetechnology.PressurizedFluid.add(<fluid:water>);
 mods.immersivetechnology.PressurizedFluid.add(<fluid:distwater>);
 mods.immersivetechnology.PressurizedFluid.add(<fluid:high_pressure_steam>);
+
+mods.immersivetechnology.HeatExchanger.addRecipe(<fluid:low_pressure_steam>*300, <fluid:low_pressure_steam>*120, <fluid:high_pressure_steam>*75, <fluid:exhaust_steam>*120, 0, 1);
+mods.immersivetechnology.HeatExchanger.addRecipe(<fluid:low_pressure_steam>*320, <fluid:preheated_water>*16, <fluid:high_pressure_steam>*80, <fluid:condensate_water>*16, 0, 1);
+mods.immersivetechnology.HeatExchanger.addRecipe(<fluid:corium_nak>*60, <fluid:condensate_water>*125, <fluid:corium_nak_hot>*60, <fluid:preheated_water>*125, 0, 1);
+mods.immersivetechnology.HeatExchanger.addRecipe(<fluid:low_pressure_steam>*5120, <fluid:high_pressure_steam>*672, <fluid:super_critical_steam>*80, <fluid:crap_steam>*672, 0, 1);
+mods.immersivetechnology.HeatExchanger.addRecipe(<fluid:low_pressure_steam>*320, <fluid:low_pressure_steam>*448, <fluid:super_critical_steam>*5, <fluid:exhaust_steam>*448, 0, 1);
+mods.immersivetechnology.HeatExchanger.addRecipe(<fluid:low_pressure_steam>*1280, <fluid:preheated_water>*224, <fluid:super_critical_steam>*20, <fluid:condensate_water>*224, 0, 1);
```

Comment:
Replacing fluid with liquid should work here just to be consistent with the other heat exchanger recipes.